LORIS_indepth.html

```diff
@@ -581,136 +297,352 @@ <h2>DICOM archives</h2>
+			</td>
+		   <td width = "50%" style="vertical-align:middle">
+			<img width="85%" src="images/Nature_open_science.png">
+	       </td></tr></table>	
+	     </section>
+            
+               	<section>
+                <h2><font color="12a3d8">What’s involved?</font></h2>
+                        <p>
+                          Longitudinal Acquisition, Storage and Curation, <font color="f27c30">Interoperability, Reproducibility</font>, Transfer, Anonymization, Security, Privacy, Ethics, APIs,
+                          Validation, Quality Control, Protocol Checking, Preprocessing, Analysis, HPC, Provenance, Ontological Standarization, <font color="f27c30">Data Harmonization</font>, Upgrades, Maintenance, Bug Fixes, User Interface, Javascript, Bootstrap, Tracking, <font color="f27c30">Extensibility</font>, Data Management, Summary Statistics, Workflows, Development, Tool Integration, <font color="f27c30">Data Sharing</font>, Download, Multi-Modal Linking, <font color="f27c30">Querying</font>, Image Processing, Visualization, Networking, System Administration, Partnerships, Funding, HR <font color="f27c30">...No big deal!</font>
+                        </p>
+        </section>
+
+		<section>
+			<img src="images/TOSI.png">
+		      </section>
+	     
+	     <section>
+	       <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5216036/" target="_blank"><img src="images/Open_science_paper.png" width="70%"></a>
+	     </section>
+            
+             <section>
+	       <h2>Cyberinfrastructure</h2>
+	       <img src="images/Cyberinfrastructure.png">			
+	     </section>
+            
+            <section>
+            <a href="CBRAIN_indepth.html#/2" target="_blank"><h2>WHAT IS CBRAIN?</h2></a>
+
+	    <table class="reveal">
+	      <tr> 
+		<td width="50%" style="vertical-align: middle;">	
+		  <a href="https://mcin-cnim.ca/technology/cbrain/" target="_blank"><img align="center" src="images/cbrain-blue.png" style="border:0; border-radius:30px;" width="90%"></a>
+		  <p><small><br>“..is an <font color="12a3d8">OPEN SOURCE</font> web-based collaborative <font color="12a3d8">High-Performance Computing (HPC)</font> platform for <font color="12a3d8">neuroimaging research</font>, connecting researchers to the <font color="12a3d8">tools</font> required to handle <font color="12a3d8">large datasets</font>.”</small></p>
+            <p><small>More on CBRAIN *HERE - link to cbrain pres*</small></p>
+		</td>
+		<td width="50%" style="vertical-align: middle">
+		  <a href="https://www.frontiersin.org/articles/10.3389/fninf.2014.00054/full" target="_blank"><img src="images/CBRAIN_paper.png" width="90%"></a>
+		</td>
+	      </tr>
+	    </table>          
+	  </section>
+            
+            <section>
+                <h2><font color="12a3d8">LORIS-CBRAIN INTEGRATION</font></h2>
+		<p>Benefits <font color="f27c30">vs</font> Challenges</p>
```

Comment:
take out "Benefits vs challenges" - makes sense in a talk but not in a slide deck

Reply:
done